test/unit/getclient-spec.ts

```diff
@@ -11,115 +11,97 @@ const expect = chai.expect;
-        expect(() => {
-            const badContext = {
-                bindings: {
-                    orchestrationClient: { id: "" },
-                },
-            };
-            getClient(badContext);
-        }).to.throw(
-            "An orchestration client function must have an orchestrationClient input binding. Check your function.json definition."
-        );
-    });
-
-    it("returns DurableOrchestrationClient if called with valid context", async () => {
-        const client = getClient(defaultContext);
-        expect(client).to.be.instanceOf(DurableOrchestrationClient);
-    });
-
-    it("returns DurableOrchestrationClient if called with V1 context", async () => {
-        const client = getClient(v1Context);
-        expect(client).to.be.instanceOf(DurableOrchestrationClient);
-    });
-
-    describe("Azure/azure-functions-durable-js#28 patch", () => {
-        let WEBSITE_HOSTNAME: string | undefined;
-        beforeEach(() => {
-            WEBSITE_HOSTNAME = process.env.WEBSITE_HOSTNAME;
-        });
-
-        afterEach(() => {
-            process.env.WEBSITE_HOSTNAME = WEBSITE_HOSTNAME;
-        });
-
-        it("corrects API endpoints if WEBSITE_HOSTNAME environment variable not found", async () => {
-            delete process.env.WEBSITE_HOSTNAME;
-
-            const badContext = {
-                bindings: {
-                    starter: TestUtils.createOrchestrationClientInputData(
-                        TestConstants.idPlaceholder,
-                        "http://0.0.0.0:12345",
-                        defaultTaskHub,
-                        defaultConnection
-                    ),
-                },
-            };
-
-            const client = getClient(badContext);
-
-            const expectedUniqueWebhookOrigins: string[] = [Constants.DefaultLocalOrigin];
-            expect(client.uniqueWebhookOrigins).to.deep.equal(expectedUniqueWebhookOrigins);
-        });
-
-        it("corrects API endpoints if WEBSITE_HOSTNAME environment variable is 0.0.0.0", async () => {
-            // Azure Functions Core Tools sets WEBSITE_HOSTNAME to 0.0.0.0:{port} on startup
-            process.env.WEBSITE_HOSTNAME = "0.0.0.0:12345";
-
-            const badContext = {
-                bindings: {
-                    starter: TestUtils.createOrchestrationClientInputData(
-                        TestConstants.idPlaceholder,
-                        `http://${process.env.WEBSITE_HOSTNAME}`,
-                        defaultTaskHub,
-                        defaultConnection
-                    ),
-                },
-            };
-
-            const client = getClient(badContext);
-
-            const expectedUniqueWebhookOrigins: string[] = [Constants.DefaultLocalOrigin];
-            expect(client.uniqueWebhookOrigins).to.deep.equal(expectedUniqueWebhookOrigins);
-        });
-    });
+    // const defaultTaskHub = "TestTaskHub";

```

Comment:
I commented them out because currently I get build errors for bad types when I uncomment them. Fixing the types would entail about as much work as fixing the tests altogether, which I'd rather do in a separate PR as this one is already getting too big

Reply:
That makes sense, although I'm still not a fan of merging commented code. How about we delete it for now? It'll still be in the git history and looks like you already have the issue to add it back.

Reply:
Yeah that's fair. I removed the commented code